python/ray/data/impl/table_block.py

```diff
@@ -129,6 +129,9 @@ class TableBlockAccessor(BlockAccessor):
     def __init__(self, table: Any):
         self._table = table
 
+    def get_block_data(self) -> Block:
```

Comment:
How about just `get_block` or a property `block`?

Reply:
@kfstorm I can do that, I'll change function name to get_block().